dmsrc/main.dm

```diff
@@ -19,21 +19,25 @@
 /* This comment bypasses grep checks */ /var/__rust_g
 
 /proc/__detect_rust_g()
+	var/__arch_suffix = null

```

Comment:
Underscore prefixing is not necessary within the scope of an underscore prefixed function

Reply:
fair